lsf_client/lsf_client.py

```diff
@@ -7,10 +7,14 @@ class LSFClient(object):
 
     def submit(self, command, stdout):
         bsub_command = ['bsub', '-sla', settings.LSF_SLA, '-oo', stdout]
+        toil_lsf_args = '-sla %s' % settings.LSF_SLA
```

Comment:
Should we put -sla in both command line and in `TOIL_LSF_ARGS`? Or just one place is fine?

Reply:
I was keeping it separate because the leader job might have different options like -oo [stdout path]